### PR TITLE
Beta Release 0.10.2

### PR DESCRIPTION
**Pull request by SiravitPhokeed**
## Features
- Changed Account Not Found Dialog verbiage to match reality
  Resolves FRO-129
- Feedback on successful enrollment
  Resolves FRO-119
- Added analytics to Name Change Dialog

## Code Changes
- Moved Account Not Found Dialog from `_app.tsx` to Landing page

## Fixes
- ”Enroll in Elective” Custom Event fired even when enrollment failed
  Fixes FRO-130
- Hide Learn Elective Entry Card if no Electives available
  Fixes FRO-106
- Hide Learn Electives page if no Electives available
- On mobile, Classes Details Dialog opens immediately on visit to Classes page without any interaction

## Preview
Preview available at https://preview.mysk.school/.

Resolves FRO-133 [(link)](https://linear.app/skiso/issue/FRO-133/beta-release-0102)

### CODE REVIEW

File: public/static/locales/en-US/landing.json [hunk 1]
```diff
@@ -22,6 +22,13 @@
   },
   "credits": "MySK is a project by SK IT Solutions • Supervised by Supannee Supeerath and Atipol Sukrisadanon • Design and development by Siravit Phokeed and Smart Wattanapornmongkol • Quality assurance by Tempoom Leelacharoen • Translations to Thai contributed by Suankularb English Club",
   "dialog": {
+    "accountNotFound": {
+      "title": "Account not found",
+      "desc": "There might be a problem with your account. If you are a current student, please report this issue via <0>this form</0>.",
```
Comment (pixelpxed) [inline public/static/locales/en-US/landing.json:27]: Just asking, what is this <0> element here?

**Review by pixelpxed — APPROVED** [review]
LGTM!

**Review by Sun-52 — APPROVED** [review]
LGTM

File: public/static/locales/en-US/landing.json (hunk 1 above)
Comment (SiravitPhokeed) [reply public/static/locales/en-US/landing.json:27]: [Replacement tags](https://react.i18next.com/latest/trans-component#how-to-get-the-correct-translation-string). `next-i18next` will use the first (0-index) component passed into `Trans` to replace `<0></0>`.

**Review by CursorWind — APPROVED** [review]
LGTM

**Pull request merged**